tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeSpacetimeQuantities.cpp

```diff
@@ -0,0 +1,78 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/range/combine.hpp>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+template <typename Symmetry, typename IndexList>
+void check_tensor_doubles_equals_tensor_datavectors(
+    const Tensor<DataVector, Symmetry, IndexList>& tensor_dv,
+    const Tensor<double, Symmetry, IndexList>& tensor_double) {
+  const size_t n_pts = tensor_dv.begin()->size();
+  for (decltype(auto) datavector_and_double_components :
+       boost::combine(tensor_dv, tensor_double)) {
+    for (size_t s = 0; s < n_pts; ++s) {
+      CHECK(boost::get<0>(datavector_and_double_components)[s] ==
+            boost::get<1>(datavector_and_double_components));
+    }
+  }
+}
+
+template <typename DataType>
+auto make_lapse(const DataType& datavector_for_size) {
```

Comment:
These are also called with DataType=double, in which case `datavector_for_size` is a very strange name.  `make_with_value` calls the argument `input`, which is also not great.  I suggest something like `structure`.